tests/eas/acceptance.py

```diff
@@ -383,17 +291,27 @@ def test_big_cpus_fully_loaded(self):
 
             self.assertGreater(busy_time, expected_busy_time, msg=msg)
 
-    def test_little_cpus_run_tasks(self):
+    @experiment_test
+    def test_little_cpus_run_tasks(self, experiment, tasks):
         """Offload Migration and Idle Pull: Little cpus run tasks while bigs are busy"""
-        tasks = self.params.keys()
+
         num_offloaded_tasks = len(tasks) / 2
 
-        first_task_finish_time = min(self.end_times.values())
+        end_times = self.calculate_end_times(experiment).values()
+        first_task_finish_time = min(end_times)
+
+        m_assert = self.get_multi_assert(experiment, "migrators")
+        start_time = min(t['starttime']
+                         for t in m_assert.getStartTime().itervalues())
+        migrator_activation_time = start_time + OFFLOAD_MIGRATION_MIGRATOR_DELAY
+
+        window = (migrator_activation_time, first_task_finish_time)
+
+        a_assert = self.get_multi_assert(experiment)
 
-        window = (self.get_migrator_activation_time(), first_task_finish_time)
-        busy_time = self.a_assert.getCPUBusyTime("cluster",
-                                                 self.env.target.bl.littles,
-                                                 window=window)
+        busy_time = a_assert.getCPUBusyTime("cluster",
```

Comment:
Again, these last two chunks seems more an update of the test code instead of a refactoring to be an EasTest.
Can we move them in a following patch?